### DISCUSSION TITLE
ADR Proposals as discussions

### DISCUSSION THREAD

**Discussion by bderusha** [Green-Software-Foundation/discussions#80] [Carbon Aware SDK]
What do people think of the idea of moving ADR proposals and their subsequent conversation to discussion threads?

I was thinking that it may be a better way to organize and talk about the ideas. The contents of the ADR would remain the same. The only difference would be that a PR capturing the ADR in the repo would be delayed until the discussion concludes and a final decision is made here in the discussion threads.

**Comment by NAMRATA-WOKE** [comment]
@bderusha is there an issue to provide more context on the ADR proposals?

**Reply by bderusha** [reply]
The project uses Architectural Decision Records ([detailed here](https://github.com/Green-Software-Foundation/carbon-aware-sdk/blob/dev/docs/architecture/decisions/0001-record-architecture-decisions.md)) to track and discuss major decisions.

However there has been a mix of using issues ([like this one](https://github.com/Green-Software-Foundation/carbon-aware-sdk/issues/351)) and pull requests ([like this](https://github.com/Green-Software-Foundation/carbon-aware-sdk/pull/261)) to open up the proposals for public comment. Each with their own pros and cons. I wonder if using this `Discussions` feature might be a better place to discuss and decide on such changes.
